cg/services/fastq_file_service/fastq_file_service.py

```diff
@@ -0,0 +1,27 @@
+from pathlib import Path
+
+from cg.services.fastq_file_service.utils import (
+    concatenate_forward_reads,
+    concatenate_reverse_reads,
+    remove_raw_fastqs,
+)
+
+
+class FastqFileService:
+
+    def concatenate(
+        self,
+        fastq_directory: Path,
+        forward_output: Path,
+        reverse_output: Path,
+        remove_raw: bool = False,
+    ):
+        concatenate_forward_reads(directory=fastq_directory, output_file=forward_output)
```

Comment:
Consider generating a unique file name in these calls, storing it and renaming them to the correct names after removing the raw fastqs

Reply:
Which files are we afraid of overwriting? The ones we are concatenating?